### PR TITLE
Associate the asset tool with a phase

### PR DESCRIPTION
**Pull request by smcalilly**
## Overview
See title. For issue #94. I also changed the LocalAsset.phase relationship to a `ForeignKey` instead of a `ManyToManyField`.

### Demo
phase-assets

### Notes
This PR is still using the prototype's map UX. Those changes are to come with the relevant issues.

## Testing Instructions
- Visit review app: https://ccfp-asset-d-phase-asse-ffhjgx.herokuapp.com/
- Login with the "ccfp-asset-dashboard test user" credentials in the LastPass shared folder.
- Select a project and go to the phases page.
- Select a phase.
- Click "edit assets" in the top left corner.
- Search for some assets. Save them.
- Make sure the assets load.
- Leave the page and come back via the phase edit page. Make sure your saved assets load in the map.

### CODE REVIEW

File: asset_dashboard/views.py [hunk 1]
```diff
@@ -275,17 +275,23 @@ class AssetAddEditView(LoginRequiredMixin, TemplateView):
...
         context = super().get_context_data(**kwargs)
 
-        context['project'] = Project.objects.get(id=self.kwargs['pk'])
+        phase = Phase.objects.get(id=self.kwargs['pk'])
+
+        context.update({
+            'phase': phase,
+            'project': Project.objects.filter(phases=phase)[0],
+            'props': {
+                'phase_id': phase.id
```
Comment (smcalilly) [inline asset_dashboard/views.py:284]: need this in `context['props']` so we can access the phase id in the react component.

File: asset_dashboard/serializers.py [hunk 2]
```diff
@@ -92,6 +93,10 @@ def get_geom(self, obj):
 class LocalAssetWriteSerializer(BaseLocalAssetSerializer):
     geom = GeometryField()
 
+    def create(self, validated_data):
+        phase = validated_data.pop('phase')
```
Comment (fgregg) [inline asset_dashboard/serializers.py:97]: why do you need this? why doesn't

```
LocalAsset.objects.create(**validated_data)
```

not just work

**Review by fgregg — CHANGES_REQUESTED** [review]
looking very clean! do have one question.

File: asset_dashboard/serializers.py (hunk 2 above)
Comment (smcalilly) [reply asset_dashboard/serializers.py]: good question. that was left over from working through this code. it works without overriding the create method, changed here: bf1976c

**Review by fgregg — APPROVED** [review]

**Pull request merged**